src/storage/src/source/reclock.rs

```diff
@@ -176,6 +193,43 @@ impl ReclockFollower {
         Ok(dest_frontier)
     }
 
+    /*
+    /// Returns an iterator of timestamp bindings for a given partition
+    fn partition_bindings(&self, pid: &PartitionId) -> Ref<PartitionBindings> {
+        Ref::map(RefCell::borrow(&self.inner), |inner| {
+            &inner.partition_bindings(pid)
+        })
+    }*/
+
+    /// Compacts the internal state
```

Comment:
Why is this "outside" and not on the inner?

Reply:
Its an api we eventually want to be available on the outside, I believe